data/mods/Medieval_Stuff/armor.json

```diff
@@ -58,7 +58,7 @@
         "encumbrance" : 10,
         "bashing" : 10,
         "flags" : ["VARSIZE", "STURDY"],
-        "coverage" : 80,
+        "coverage" : 60,
```

Comment:
Would 70 work? Needs to be distinct from galea while still being functional as a helmet.